test/trivial_ico.js

```diff
@@ -71,4 +71,34 @@ contract('TrivialToken - ICO tests', (accounts) => {
         await contributeIco();
         await finishIco();
     })
+
+    it('cancel ICO no contributors', async () => {
```

Comment:
These test tests for two different cases. One is that `claimIcoContribution` may be only called by contributors and the other is that `contributeInIco` cannot be called in this IcoCancelled state. I would separate this into two different tests, so we can keep up with the "one test = one case" rule.